assets/custom/js/signup.js

```diff
@@ -0,0 +1,486 @@
+
+
+var referralCode = Lockr.get('REFERRAL_CODE');
+logService = LexLogService("email");
+
+var html = "<table>";
+
+if(Lockr.get("history")){
+Lockr.get("history").forEach(function(row) {
+	html += "<tr><td>" + row.question + "</td><td>" + row.answer + "</td></tr>";
+});
+}
+
+html += "</table>";
+
+
+
+
+/*
+var settings = {
+  "async": true,
+  "crossDomain": true,
+  "url": "https://api.imgur.com/3/image",
+  "type": "POST",
+  "headers": {
+    "authorization":  "Client-ID 84b67a72469fab6",
+    "authorization": "Bearer 27ccaf80d0e73825819847b4b68916ab9912d4e3"
+  },
+  "processData": false,
+  "contentType": false,
+  "mimeType": "multipart/form-data",
+  "data": formData
+}
+
+$.ajax(settings).done(function (response) {
+  console.log(response);
+});
+*/
+
+
+
+
+/*
+var formData = new FormData();
+	
+	formData.append("image", $('input[type=file]')[0].files[0]); 
+
+$.ajax({
+	dataType: 'jsonp',
+    url: 'https://getsimpleform.com/messages/ajax?form_api_token=3f5bff9511b02b28a7a9ce8690c51a67',
+    data: formData,
+    type: 'POST',
+    contentType: false, // NEEDED, DON'T OMIT THIS (requires jQuery 1.6+)
+    processData: false, // NEEDED, DON'T OMIT THIS
+    // ... Other options like success and etc
+});
+*/
+
+
+
+//formData.append("username", "Groucho");
+//formData.append("accountnum", 123456); // number 123456 is immediately converted to a string "123456"
+
+// HTML file input, chosen by user
+
+
+
+//var request = new XMLHttpRequest();
+//request.open("POST", "https://getsimpleform.com/messages?form_api_token=3f5bff9511b02b28a7a9ce8690c51a67");
+//request.send(formData);
+
+//function to submit the form to simple form site
+//not doing ajax allows to submit the picture as attachment
+function submitSimpleForm(){
+	document.getElementById("attorney-signup-form").action = "https://getsimpleform.com/messages?form_api_token=3f5bff9511b02b28a7a9ce8690c51a67";
+	document.getElementById("attorney-signup-form").submit();
+}
+
+
+//function to make post request to convertkit 
+//works, but no response given which should be fine
+function convertkitPost(area, name, emailIn) {
+  // Add the iframe with a unique name
+  var iframe = document.createElement("iframe");
+  var uniqueString = "Tz4krNAWrdBuLkKO7v3U";
+  document.body.appendChild(iframe);
+  iframe.style.display = "none";
+  iframe.contentWindow.name = uniqueString;
+
+  var attorney_id = "217520";
+  var dui_id = "215927";
+  var divorce_id = "217162";
+  var business_id = "217163";
+
+	var tag_id = attorney_id;
+
+
+  //i know this is stupid as hell
+  //wehen we get more branches ill redo this
+
+	var tags_final;
+
+  if (area == '1')
+	{
+	  tags_final = "215927";
```

Comment:
It will be easier to understand this code in the future if we use the variable name instead of hardcode int

e.g. tags_final = dui_id + ', ' + divorce_id